Libraries/Network/NetInfo.js

```diff
@@ -79,56 +80,51 @@ const _isConnectedSubscriptions = new Map();
  * NetInfo exposes info about online/offline status
  *
  * ```
- * NetInfo.fetch().then((reach) => {
- *   console.log('Initial: ' + reach);
+ * NetInfo.getConnection().then((connection) => {
```

Comment:
This name is quite confusing (because it is not connection), can/should we rename it to something like `getConnectionInfo` or `getConnectionStatus`?